data/interfaces/default/newsletter_config.html

```diff
@@ -470,9 +485,207 @@ <h4 class="modal-title" id="newsletter-config-modal-header">${newsletter['agent_
     </div>
 </div>
 
-<script src="${http_root}js/jquery-cron-min.js"></script>
+        <script src="${http_root}js/jquery-cron-min.js"></script>
+        <script src="${http_root}js/ace.js"></script>
+        <script src="${http_root}js/mode-html.js"></script>
+        <script src="${http_root}js/theme-tautulli.js"></script>
+        <script src="${http_root}js/beautify-html.min.js"></script>
+
 <script>
 
+    // Initialize Ace Editor for the message field
+    var messageEditor;
+    
+    function initMessageEditor() {
+        if (messageEditor) {
+            messageEditor.destroy();
+        }
+        
+        try {
+            // Check if Ace editor is available
+            if (typeof ace === 'undefined') {
+                throw new Error('Ace editor not loaded');
+            }
+            
+            messageEditor = ace.edit('message-editor');
+            messageEditor.setTheme('ace/theme/tautulli');
+            messageEditor.session.setMode('ace/mode/html');
+            messageEditor.setOptions({
+                showPrintMargin: false,
+                showGutter: true,
+                fontSize: '14px',
+                tabSize: 2,
+                wrap: true,
+                highlightActiveLine: true
+            });
+            
+            // Set initial value from the hidden textarea
+            var initialValue = $('#message').val() || '';
+            messageEditor.setValue(initialValue);
+            messageEditor.clearSelection();
+            
+            // Sync Ace editor content with hidden textarea on change
+            messageEditor.getSession().on('change', function() {
+                $('#message').val(messageEditor.getValue());
+            });
+            
+            // Resize editor to fit content
+            messageEditor.resize();
+            
+            // Set up event handlers for editor controls
+            setupEditorControls();
+            
+        } catch (error) {
+            console.error('Failed to initialize Ace editor:', error);
+            // Fallback to regular textarea if Ace editor fails
+            $('#message-editor').hide();
+            $('#message').show().removeAttr('style').css({
+                'width': '100%',
+                'height': '300px',
+                'border': '1px solid #444',
+                'border-radius': '4px',
+                'background-color': '#333',
+                'color': '#eee',
+                'font-family': 'Monaco, Menlo, Ubuntu Mono, monospace',
+                'font-size': '14px',
+                'padding': '10px'
+            });
```

Comment:
Also need to hide the editor controls.

Removed CSS styles that are already part of the textarea.

```suggestion
            $('.editor-controls').hide();
            $('.editor-container').css({
                'border': 'none',
            });
            $('#message').show().removeAttr('style').css({
                'height': '300px',
                'font-family': 'Monaco, Menlo, Ubuntu Mono, monospace'
            });
```